build.cmd

```diff
@@ -470,6 +474,7 @@ if /i "%BUILD_CORECLR%" == "1" (
 
 if /i "%BUILD_FROMSOURCE%" == "1" (
     set NEEDS_DOTNET_CLI_TOOLS=1
+    set COPY_FSCOMP_RESOURCE_FOR_BUILD_FROM_SOURCES=1
```

Comment:
No - do this for the **non-build-from-source** cases.  The copy is intended to prepare the generated FSComp files for checkin when you do something like `build net40`

Reply:
I'm not sure about this particular line, but this change to build.cmd actually seems to makes the CI pass the "build from source" test without manually having to take care of those files, I'm probably missing something but ideally that is what should happen without any intervention from contributors.

Unless for some reason, we'd have different messages going there.

Should I remove this line and check that it is still passing the build from source test?

Reply:
I've removed that line from #5119, I'll track there that whole change for build-from-source fscomp files.

Reply:
@smoothdeveloper Don, has this correct, when building net40, copy over the files.  until we can fix the  redhat build.